org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/server/Server.java

```diff
@@ -106,18 +106,19 @@ public static String getSonarCloudUrl() {
   Server(String id) {
     this.id = id;
 
-    Set<String> excludedPlugins = new HashSet<>(Arrays.asList("typescript", "java", "cpp"));
+    EnumSet<Language> languagesDisabledByDefault = EnumSet.of(Language.TS, Language.JAVA, Language.CPP);
```

Comment:
I think at some point we should plan to disable Cobol by default. But for that we would have to require our partner to whitelist Cobol on their side. Maybe we could create a ticket to not forget.

Reply:
https://jira.sonarsource.com/browse/SLE-341